### DISCUSSION TITLE
PaddleOCR/model/index

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13919] [Q&A]
# PaddleOCR/model/index

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/model/index.html

**Comment by 111395991161ysm** [comment]
做得和ultralytics的yolov10一样了，这样用起来的确方便，结构清晰，方便查找，爆赞。
